### PR TITLE
Some language improvements

### PR DESCRIPTION
**Pull request by rumpelsepp**

### CODE REVIEW

**Comment by rumpelsepp** [comment]
Seriously? Please improve this test suite; it's quite frustrating. My language fixes cannot in any case cause a database deadlock. I already ranted against the test suite here: https://github.com/gitlabhq/gitlabhq/pull/9065#issuecomment-94727091

```
Pending:
  Gitlab::Markdown::UserReferenceFilter links with adjacent text
    # TODO (rspeicher): Re-enable when usernames can't end in periods.
    # ./spec/lib/gitlab/markdown/user_reference_filter_spec.rb:67

Failures:

  1) Comments On a merge request the note form with text should have enable submit button and preview button
     Failure/Error: Unable to find matching line from backtrace
     ActiveRecord::StatementInvalid:
       PG::Error: ERROR:  deadlock detected
       DETAIL:  Process 6849 waits for AccessShareLock on relation 127446 of database 16507; blocked by process 6414.
       Process 6414 waits for AccessExclusiveLock on relation 127512 of database 16507; blocked by process 6849.
       HINT:  See server log for query details.
       : SELECT "namespaces"."id" FROM "namespaces" INNER JOIN "members" ON "namespaces"."id" = "members"."source_id" WHERE "members"."type" IN ('GroupMember') AND "members"."source_type" = 'Namespace' AND "namespaces"."type" IN ('Group') AND "members"."user_id" = $1  ORDER BY "namespaces"."created_at" DESC, "namespaces"."id" DESC, "members"."created_at" DESC, "members"."id" DESC
     # ./app/models/user.rb:289:in `authorized_groups'
     # ./app/services/projects/participants_service.rb:41:in `groups'
     # ./app/services/projects/participants_service.rb:11:in `execute'
     # ./app/controllers/projects_controller.rb:118:in `autocomplete_sources'
     # ./lib/gitlab/middleware/static.rb:9:in `call'

2 deprecations found.

Finished in 8 minutes 16 seconds
3420 examples, 1 failure, 1 pending

Failed examples:

rspec ./spec/features/notes_on_merge_requests_spec.rb:37 # Comments On a merge request the note form with text should have enable submit button and preview button
rake aborted!
["rspec", "spec"] failed!
/home/runner/gitlabhq/lib/tasks/spinach.rake:38:in `block in run_commands'
/home/runner/gitlabhq/lib/tasks/spinach.rake:37:in `each'
/home/runner/gitlabhq/lib/tasks/spinach.rake:37:in `run_commands'
/home/runner/gitlabhq/lib/tasks/spec.rake:38:in `block in <top (required)>'
Tasks: TOP => test_ci => spec
(See full trace by running task with --trace)
```

**Comment by jvanbaarsen** [comment]
@rumpelsepp We are aware that the test suite is a bit buggy at the moment. I've restarted the build for you.

File: app/views/projects/merge_requests/show/_mr_accept.html.haml [hunk 1]
```diff
@@ -40,12 +40,12 @@
...
     %p
       %span
-        %strong This repository does not have satellite. Ask an administrator to fix this issue
+        %strong This repository does not have a satellite. Please, ask an administrator to fix this issue!
 
   .automerge_widget.cannot_be_merged.hide
     %h4
-      This request can't be merged with GitLab.
-      You should do it manually with
+      This request cannot be merged with automatically.
```
Comment (jvanbaarsen) [inline app/views/projects/merge_requests/show/_mr_accept.html.haml]: I don't think this line is correct. What about:
"This request cannot be merged automatically with GitLab."?

**Comment by rumpelsepp** [comment]
> We are aware that the test suite is a bit buggy at the moment. I've restarted the build for you.

Thanks. :) And sorry for the rant.

File: app/views/projects/merge_requests/show/_mr_accept.html.haml (hunk 1 above)
Comment (rumpelsepp) [inline app/views/projects/merge_requests/show/_mr_accept.html.haml]: Oh, yeah. You're right. :blush:

**Comment by jvanbaarsen** [comment]
@rumpelsepp No problem :D Its a big annoyance, so I get why you get annoyed about it :) (I do so too myself :) ). But its a hard problem to solve, since we are running tests in parallel with javascript. that does give certain problems :( But it's on our radar, and we are constantly trying to improve in the test suite. (If you compare it to 6 - 12 months ago, its heavily improved!).

So long story short, its annoying, its on our radar, we are trying to fix it :dancer: 

thanks for all your great contributions lately :) :heart:

File: app/views/projects/merge_requests/show/_mr_accept.html.haml [hunk 2]
```diff
@@ -40,12 +40,12 @@
   .automerge_widget.no_satellite.hide
     %p
       %span
-        %strong This repository does not have satellite. Ask an administrator to fix this issue
+        %strong This repository does not have a satellite. Please, ask an administrator to fix this issue!
```
Comment (tsigo) [inline app/views/projects/merge_requests/show/_mr_accept.html.haml]: Comma after "Please" is unnecessary.

File: app/views/projects/merge_requests/show/_mr_accept.html.haml [hunk 3]
```diff
@@ -1,12 +1,12 @@
 - unless @allowed_to_merge
   - if @project.archived?
     %p
-      %strong Archived projects cannot be committed to!
+      %strong Archived projects do not provide commit access, sorry.
```
Comment (tsigo) [inline app/views/projects/merge_requests/show/_mr_accept.html.haml]: I think this should be "Archived projects do not provide commit access." without the apology.

Comment (rumpelsepp) [inline app/views/projects/merge_requests/show/_mr_accept.html.haml]: thx. I will update this pull request ASAP.

File: app/views/projects/merge_requests/show/_mr_accept.html.haml (hunk 2 above)
Comment (rumpelsepp) [inline app/views/projects/merge_requests/show/_mr_accept.html.haml]: thx. I will update this pull request ASAP.

**Comment by rumpelsepp** [comment]
Just wanted to update this pull request. There are some bugs in current master which don't allow creating a new user. So I can't acutally test how these changes look like. I'll wait a bit until its fixed and update this pull request again. :)

**Comment by jvanbaarsen** [comment]
@rumpelsepp Can you rebase with master so we can merge this?

**Comment by rumpelsepp** [comment]
Here you go. Please review. :)

**Comment by jvanbaarsen** [comment]
@rumpelsepp Thanks!

**Pull request merged**